<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferFunctionClass</class>
 <widget class="QWidget" name="TransferFunctionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>312</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>312</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>312</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Transfer Function</string>
  </property>
  <widget class="Display" name="DisplayR" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>258</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="Display" name="DisplayG" native="true">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>258</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="Display" name="DisplayB" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>258</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="Display" name="DisplayA" native="true">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>258</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Display</class>
   <extends>QWidget</extends>
   <header>Display.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
